Lesson_3_HW/src/main/java/GU_Android_1089/Lesson_3/HW/Homework_Practice_GuessNumberGame.java

```diff
@@ -0,0 +1,99 @@
+package GU_Android_1089.Lesson_3.HW;
+
+import javax.xml.namespace.QName;
+import java.util.Scanner;
+
+public class Homework_Practice_GuessNumberGame {
+
+    public static void main(String[] args) {
+        // Задание 1. Написать программу, которая загадывает случайное число от 0 до 9 и пользователю дается 3 попытки
+        // угадать это число. При каждой попытке компьютер должен сообщить, больше ли указанное пользователем число,
+        // чем загаданное, или меньше. После победы или проигрыша выводится запрос – «Повторить игру еще раз?
+        // 1 – да / 0 – нет»(1 – повторить, 0 – нет).+
+        init();
+    }
+
+    /**
+     * This method is used to start the game
+     */
+    private static void init(){
+        int hiddenNumber = randomNumberGenerator();
+        System.out.println("Загадано цело число от 0 до 9. Отгадаете ли Вы его за 3 попытки? Игра началась!");
+        CompareNumbers(hiddenNumber);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.